src/windowing.js

```diff
@@ -1,17 +1,17 @@
 export function blackman(size) {
   let blackmanBuffer = new Float32Array(size);
-  let coeff1 = 2 * Math.PI / (size - 1);
+  let coeff1 = (2 * Math.PI) / (size - 1);
   let coeff2 = 2 * coeff1;
 
   //According to http://uk.mathworks.com/help/signal/ref/blackman.html
   //first half of the window
   for (let i = 0; i < size / 2; i++) {
-    blackmanBuffer[i] = 0.42 - 0.5 * Math.cos(i * coeff1) +
-            0.08 * Math.cos(i * coeff2);
+    blackmanBuffer[i] =
+      0.42 - 0.5 * Math.cos(i * coeff1) + 0.08 * Math.cos(i * coeff2);
   }
 
   //second half of the window
-  for (let i = size / 2; i > 0; i--) {
+  for (let i = Math.ceil(size / 2); i > 0; i--) {
```

Comment:
Here is the important change, the rest is all formatting. Prettier strikes again!